### PR TITLE
Add windows init file

### PR DESCRIPTION
**Pull request by NerijusRazvodovskis**
Related to https://hostingers.atlassian.net/browse/INFRA-21

### CODE REVIEW

File: nebula_windows.go [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+	"go.uber.org/zap"
+)
+
+func nebulaDownloadLink() string {
+	return "https://github.com/hostinger/packages/releases/download/v1.0.0/nebula-1.5.2-windows-amd64.exe"
+}
+
+// NebulaDir absolute paths to the directory of Nebula configurations and binaries
+func NebulaDir() string {
+	return "C:/Nebula"
```
Comment (ton31337) [inline nebula_windows.go:19]: Is this the best path for user programs?

File: nebula_windows.go [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+}
+
+// NebulaExecutable show full path of Nebula executable
+func NebulaExecutable() string {
+	return path.Join(NebulaDir(), "nebula")
+}
+
+func getDefaultGateway(x string) (string, error) {
+	args := x
```
Comment (ton31337) [inline nebula_windows.go:28]: Any reason for this assignment?

File: nebula_windows.go [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+	var defaultGateway string
+	lines, err := exec.Command("powershell", "Find-NetRoute", "-RemoteIPAddress", args).CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+	for _, param := range strings.Split(string(lines), "\n") {
+		if !strings.Contains(param, "NextHop") {
+			continue
+		}
+		defaultGateway = strings.TrimSpace(strings.Split(param, ":")[1])
```
Comment (ton31337) [inline nebula_windows.go:38]: You should check if the split didn't fail, otherwise, you will have unknowns.

File: nebula_windows.go [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+	}
+	return defaultGateway, nil
+}
+
+func getIfIndex(x string) (string, error) {
```
Comment (ton31337) [inline nebula_windows.go:43]: This is the same as getDefaultGateway(), I'd create a separate struct for storing all the gateway related info inside.

File: nebula_windows.go [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+	ifIndex, err := getIfIndex(e.RemoteIP)
+	if err != nil {
+		Cfg.Logger.Error("can't get ifIndex",
+			zap.Error(err))
+		return err
+	}
+
+	ns := strings.Join(nameServers, ",")
+	cmd_dns := exec.Command("powershell", "Set-DnsClientServerAddress", "-InterfaceIndex", ifIndex, "-ServerAddresses", ns)
+	cmd_dns_suffix := exec.Command("powershell", "Set-DnsClient", "-InterfaceIndex", ifIndex, "-ConnectionSpecificSuffix", DNSAutoDiscoverZone)
```
Comment (ton31337) [inline nebula_windows.go:113]: Use camelCase style for variables everywhere.

File: nebula_windows.go [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+	}
+
+	index, err := getIfIndex(e.RemoteIP)
+	if err != nil {
+		Cfg.Logger.Error("can't get ifIndex",
+			zap.Error(err))
+		return err
+	}
+
+	if err := exec.Command("powershell", "route", "delete", e.RemoteIP, "mask", "255.255.255.255", gw, "if", index).Run(); err != nil {
```
Comment (ton31337) [inline nebula_windows.go:152]: Maybe Windows has `replace` method?

File: nebula_windows.go (hunk 1 above)
Comment (dominykasn) [reply nebula_windows.go:19]: Probably C:\Program Files\Nebula would be a better location if you don't have any directories already in mind, @NerijusRazvodovskis ?

Comment (NerijusRazvodovskis) [reply nebula_windows.go:19]: indeed 👍🏻

File: nebula_windows.go (hunk 2 above)
Comment (NerijusRazvodovskis) [reply nebula_windows.go:28]: Yep not really, will remove it.

**Comment by dominykasn** [comment]
There are also some libraries like https://github.com/KnicKnic/go-powershell which allow to run poweshell commands without command.exec, probably would be easier for you to use instead of calling the command method everytime

**Pull request merged**